source/includes/inkling-reference/_concepts.html.md

```diff
@@ -1,8 +1,23 @@
-# Concept
+# Concepts
 
-Reference for the keyword **concept**. Also, describes the keywords: **predicts**, **input**, **output**, **is**, **follows**, **end**, and **feeds**.
+The `concept` keyword declares an abstract concept that is to be learned by the system. Ultimately, this takes the form of a transformation of data, but no information need be provided about how to perform the calculation. By declaring a concept, you are instructing the BRAIN server that this is a node in the basic recurrent artificial intelligence network that must be learned. Consequently, concept nodes must have corresponding curricula to teach them.
+
+Because concepts are learned, their declarations tend to be fairly simple unless one wants to explicitly tell the BRAIN server what learning algorithms and architecture to use (which is an unusual case).  A typical statement will look something like this:
+
+```inkling--code
+concept AbstractConceptName
+  is classifier
+  predicts ConceptSchema
+  follows Antecedent1, Antecedent2      // 'follows' concept, block, or stream
+  feeds Dependent1                      // 'feeds' concept, block, or stream
+end
+```
 
-### What is it?
+The `follows` and `feeds` keywords establish connectivity in the BRAIN directed graph in the same way that the `from` and `into` keywords do in stream declarations. The `is` keyword specifies the overall class of concept that is being modeled. For example, a classifier will learn to identify a label corresponding to its input, an estimator will learn to predict a value, a predictor will learn sequences and predict subsequent items in a sequence, etcetera.
```

Comment:
first sentence should be:
The `follows` and `feeds` keywords establish connectivity in the BRAIN directed graph.